frontend/default/routes.py

```diff
@@ -14,29 +14,14 @@ def index():
 
 @default_bp.errorhandler(404)
 def not_found(error):
-    current_app.logger.error(f"Encountered 404: {error}")
-    round_data = get_round_data(Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID, as_dict=True)
-    return render_template(
-        "404.html",
-        contact_us_phone=round_data.contact_details["phone"],
-        contact_us_email_address=round_data.contact_details["email_address"],
-        contact_us_text_phone=round_data.contact_details["text_phone"],
-        opening_time=round_data.support_availability["time"],
-        opening_days=round_data.support_availability["days"],
-        closed=round_data.support_availability["closed"]
-    ), 404
+    current_app.logger.warning(f"Encountered 404: {error}")
+    round_data = get_round_data_fail_gracefully(
+        Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID
+    )
+    return render_template("404.html", round_data=round_data), 404
 
 
 @default_bp.errorhandler(500)
 def internal_server_error(error):
     current_app.logger.error(f"Encountered 500: {error}")
-    round_data = get_round_data(Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID, as_dict=True)
-    return render_template(
-        "500.html",
-        contact_us_phone=round_data.contact_details["phone"],
-        contact_us_email_address=round_data.contact_details["email_address"],
-        contact_us_text_phone=round_data.contact_details["text_phone"],
-        opening_time=round_data.support_availability["time"],
-        opening_days=round_data.support_availability["days"],
-        closed=round_data.support_availability["closed"]
-    ), 500
+    return render_template("500.html"), 500
```

Comment:
Looks the same as in frontend - can we move all these error handling routes into utils and then any apps with a frontend will have the same behaviour/content for an error?